components/script/dom/gpucommandbuffer.rs

```diff
@@ -20,31 +24,54 @@ pub struct GPUCommandBuffer {
     channel: WebGPU,
     label: DomRefCell<Option<DOMString>>,
     command_buffer: WebGPUCommandBuffer,
+    buffers: DomRefCell<HashSet<Dom<GPUBuffer>>>,
 }
 
 impl GPUCommandBuffer {
-    pub fn new_inherited(channel: WebGPU, command_buffer: WebGPUCommandBuffer) -> GPUCommandBuffer {
+    #[allow(unrooted_must_root)]
+    pub fn new_inherited(
+        channel: WebGPU,
+        command_buffer: WebGPUCommandBuffer,
+        buffers: HashSet<Dom<GPUBuffer>>,
```

Comment:
Let's accept `HashSet<DomRoot<GPUBuffer>>` instead and remove the unrooted_must_root annotation.